apps/ui/src/components/Ui/ContainerSettings.vue

```diff
@@ -6,7 +6,7 @@ defineProps<{
 </script>
 
 <template>
-  <div class="mb-4">
+  <div class="mb-4 mx-4 max-w-[592px] flex-grow pb-[100px]">
```

Comment:
@ChaituVR can you explain why was `mx-4` added there?

It caused this issue: https://github.com/snapshot-labs/workflow/issues/475

I removed it and it seems to solve the issue, but not sure if it breaks anything else 🤔

Reply:
It will break settings page if we remove this, previously it was on SpaceSettings but I had to move it to down because of the bottom navbar with "Save", didn't realize we use same on space creation

Reply:
Something like this work https://github.com/snapshot-labs/sx-monorepo/pull/1237